### DISCUSSION TITLE
Turn of auto retry fail mutation due to internet-recover after offline

### DISCUSSION THREAD

**Discussion by tunguyen-ct** [TanStack/query#5101] [Q&A]
Is there any option I can use to completely turn off this behavior?

https://tanstack.com/query/v3/docs/react/guides/mutations#retry

> If mutations fail because the device is offline, they will be retried in the same order when the device reconnects.

**Comment by TkDodo** [comment]
retries default to `0` so unless you specify retries, this won't happen.
